src/backend/utils/misc/guc-file.l

```diff
@@ -130,6 +131,22 @@ STRING			\'([^'\\\n]|\\.|\'\')*\'
  */
 void
 ProcessConfigFile(GucContext context)
+{
+	(void) ProcessConfigFileImpl(context, false);
+}
+
+/*
+ * This works the same way as the ProcessConfigFile(), except that it returns a
+ * list of options (config_generic *) that were changed.
+ */
+List *
+ProcessConfigFileForSync(GucContext context)
+{
+	return ProcessConfigFileImpl(context, true);
+}
```

Comment:
put these lines after ProcessConfigFileImpl and you don't need redundant declaration of this function